static/js/components/ValidationAlert.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import _ from 'lodash';
+
+import type { ValidationErrors } from '../flow/profileTypes';
+
+export default class ValidationAlert extends React.Component {
+  props: {
+    errors:   ValidationErrors,
+    children: React$Element[],
+  };
+
+  alertMessage: Function = (): React$Element|void => {
+    const { errors } = this.props;
+    if ( !_.isEmpty(errors) ) {
+      return (
+        <span className="message">
+          Please fix the errors above and try again
+        </span>
+      );
+    }
+  };
+
+  render () {
+    const { children } = this.props;
+    return (
+      <div className="validation-alert">
+        { this.alertMessage() }
+        <div className="actions">
+          { children }
+        </div>
+      </div>
```

Comment:
A key thing is that the `ValidationAlert` component (as written) can also render just find without children. So what I'm doing to get it to render on the `PersonalTab` and `PrivacyTab` is just (more or less) doing this:

```js
<ValidationAlert {...this.props} />
```

so it's not wrapping anything at all. I actually think that this approach (of a component that optionally takes children) is very idiomatic for React.

Also the reason that I'm wrapping the buttons to pass in to the `actions` prop for `Dialog` with the `ValidationAlert` component is because it makes it easier to get the layout to look the way I want, not so much because of a limitation of the dialog component. The dialog component is kind flexible in this regard, actually, since it can take a single component or an array of components.